spec/models/user_spec.rb

```diff
@@ -3,9 +3,9 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
```

Comment:
If we end up keeping the `set_sort_name` method, we should probably add a test here. I don't see one, but I might have missed it.

Reply:
Added, thanks!